adaptive_shield/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+	"manifest_version": "2.0.0",
+	"app_id": "adaptive-shield",
+	"app_uuid":"0c72bf61-1de6-4408-8a24-86f8e3413e07",
+	"display_on_public_website": true,
+	"tile": {
+      "overview": "README.md#Overview",
+      "configuration": "README.md#Setup",
+      "support": "README.md#Support",
+      "changelog": "CHANGELOG.md",
+      "description": "Track SaaS posture alerts",
+      "title": "Adaptive Shield",
+      "media": [
+        {
+        "caption": "SaaS environments posture in Adaptive Shield",
```

Comment:
Would you be able to slightly edit the images so that they match [our guidelines](https://docs.datadoghq.com/developers/marketplace/#media-carousel-requirements)? Most importantly they should have a 16:9 aspect ratio. Thanks!